core/trino-main/src/main/java/io/trino/sql/planner/planprinter/PlanPrinter.java

```diff
@@ -1458,9 +1459,22 @@ private void printStatisticAggregationsInfo(
             }
 
             for (Map.Entry<ColumnStatisticMetadata, Symbol> columnStatistic : columnStatistics.entrySet()) {
+                String aggregationName;
+                if (columnStatistic.getKey().getStatisticTypeIfPresent().isPresent()) {
+                    aggregationName = columnStatistic.getKey().getStatisticType().name();
+                }
+                else {
+                    FunctionName aggregation = columnStatistic.getKey().getAggregation();
+                    if (aggregation.getCatalogSchema().isPresent()) {
+                        aggregationName = aggregation.getCatalogSchema() + "." + aggregation.getName();
```

Comment:
This seems to be a bug:

```
        Optional<String> s1 = Optional.of("hello ");
        String s2 = "world!";
        System.out.println(s1 + s2);
```
outputs
`Optional[hello ]world!`

Reply:
good catch, thanks